src/likelihoodratiotest.jl

```diff
@@ -86,21 +103,61 @@ function Base.show(io::IO, lrt::LikelihoodRatioTest; digits=2)
     println(io, "Model Formulae")
 
     for (i, f) in enumerate(lrt.formulas)
-        println("$i: $f")
+        println(io, "$i: $f")
     end
-    cols = hcat(lrt.models.dof, lrt.models.deviance,
-                _prepend_0(lrt.tests.deviancediff),
-                _prepend_0(lrt.tests.dofdiff),
-                _prepend_0(lrt.tests.pvalues))
-
-    ct = CoefTable(
-        cols, # cols
-        ["model-dof", "deviance", "χ²", "χ²-dof", "P(>χ²)"], # colnms
-        string.(1:length(lrt.formulas)), # rownms
-        5, # pvalcol
-        3 # teststatcol
-    )
-    show(io, ct)
+
+    # the following was adapted from StatsModels#162
+    # from nalimilan Δdf = lrt.tests.dofdiff
+    Δdev = lrt.tests.deviancediff
+
+    nc = 6
+    nr = length(lrt.formulas)
+    outrows = Matrix{String}(undef, nr+1, nc)
+
+    outrows[1, :] = ["",
+                    "model-dof",
+                    "deviance",
+                    "χ²",
+                    "χ²-dof",
+                    "P(>χ²)"] # colnms
+
+
+    outrows[2, :] = ["[1]",
+                    @sprintf("%.0d", lrt.dof[1]),
+                    @sprintf("%.4f", lrt.deviance[1]),
+                    " "," ", " "]
+
+    for i in 2:nr
+        outrows[i+1, :] = ["[$i]",
+                           @sprintf("%.0d", lrt.dof[i]),
+                           @sprintf("%.4f", lrt.deviance[i]),
+                           @sprintf("%.0d", Δdf[i-1]),
```

Comment:
I get
```julia
julia> MixedModels.likelihoodratiotest(fm2, fm1)
Model Formulae
1: reaction ~ 1 + days + MixedModels.ZeroCorr((1 + days | subj))
2: reaction ~ 1 + days + (1 + days | subj)
Error showing value of type MixedModels.LikelihoodRatioTest:
ERROR: UndefVarError: Δdf not defined
```
with the stack trace pointing here.

Reply:
It appears that you accidentally incorporate the line to define `Δdf` in a comment line above.  Instead of guessing what might have occurred, I'll let you undo that it it was indeed the problem.

Other than that, it looks good to go from me - well we might want to ponder why this wasn't caught in a test.

Reply:
Oh, that looks like I hit delete right before doing git add. Grrr. I'm also surprised this wasn't caught by a test ... I might have forgotten to test the `show` method itself. I'll investigate.

Reply:
Ah, because I didn't include the file with the new tests in `runtests.jl`. That's twice in the past two weeks ....

Reply:
And what do you know, when you actually enable the tests, coverage improves....